kytos/core/config.py

```diff
@@ -88,6 +88,11 @@ def __init__(self):
                             action='store_true',
                             help="Create a kytos superuser.")
 
+        parser.add_argument('-t', '--token_expiration_minutes',
+                            action='store',
+                            help="Set time in minutes to expire authentication"
+                                  " token.")
```

Comment:
I don't think we need a command-line parameter for this.

Reply:
Resolved in the last commit.